great_expectations/rule_based_profiler/types/data_assistant_result/plot_util.py

```diff
@@ -0,0 +1,97 @@
+from abc import ABC, abstractmethod, abstractproperty
+from dataclasses import dataclass
+from typing import Optional, Union
+
+import altair as alt
+
+
+class PlotUtil(ABC):
```

Comment:
There's perhaps a better name for this 😄 suggestions are welcomed.

Reply:
Let's also block this PR until we get docstrings - I'm holding off on documentation until we deem this approach appropriate.